graylog2-web-interface/src/views/components/WidgetQueryControls.jsx

```diff
@@ -95,10 +96,12 @@ const ResetFilterButton = styled(Button)`
   vertical-align: initial;
 `;
 
+const _resetOverride = () => GlobalOverrideActions.reset().then(SearchActions.executeWithCurrentState);
```

Comment:
Instead of `executeWithCurrentState` we should use the `refresh` action here, as it is also executing search refresh hooks, which the `executeWithCurrentState` action does not do.